rustls/src/error.rs

```diff
@@ -225,7 +226,6 @@ pub enum PeerMisbehaved {
     UnsolicitedSctList,
     UnsolicitedServerHelloExtension,
     WrongGroupForKeyShare,
-    MissingPskModesExtension,
```

Comment:
Rebase artifact? The commit leads with `fixup!` and doesn't appear to be doing what it says on the tin.

Reply:
Aha, already caught. I was reviewing the stale branch :-)